ftw/slider/contents/container.py

```diff
@@ -33,5 +33,5 @@ class IContainerSchema(Schema):
     )
 
 
-class Topic(Container):
+class Container(DxContainer):
```

Comment:
Is it ok to just rename the class (implementations, breaking change)?

Reply:
I think that should be fine since it was never actually implemented (The type referenced in the FTI is Container, not Topic).

Reply:
Ok. But please also add this to the changelog. Because from now on the slider container will provide the `IContainer` interface which it didn't before this change.

Reply:
I just saw that you already did that 👍